charts/flyte-core/templates/propeller/manager.yaml

```diff
@@ -0,0 +1,135 @@
+{{- if .Values.flytepropeller.enabled }}
+{{- if .Values.flytepropeller.manager }}
+apiVersion: apps/v1
```

Comment:
flytepropeller-manager is very similar to flytepropeller deployment, Can we use one resource for both ?

Reply:
Merged the deployment templates. Let me know if you like this more, I was trying to avoid the constant "if flytepropeller-manager" statements, seems messy to me.